Strings/src/com/company/task_2/Task_9.java

```diff
@@ -0,0 +1,38 @@
+package com.company.task_2;
+
+import java.util.Scanner;
+
+public class Task_9 {
```

Comment:
disadvantage of this realization is that you iterate over the string (array) twice. you should avoid extra iteration since it can affect perfomance of bigger data. please refactor to do all in one cycle

Reply:
Полностью переделал